pkg/timestream/helper.go

```diff
@@ -47,8 +47,9 @@ func QueryResultToDataFrame(res *timestreamquery.QueryOutput) (dr backend.DataRe
 		for _, timeseriesColumn := range timeseriesColumns {
 			for _, series := range res.Rows {
 				tv := series.Data[timeseriesColumn.columnIdx].TimeSeriesValue
-				if tv == nil {
-					dr.Error = fmt.Errorf("Expecting timeseries colum at: %d", timeseriesColumn.columnIdx)
+				isNullDataPoint := series.Data[timeseriesColumn.columnIdx].NullValue
+				if tv == nil && !*isNullDataPoint {
```

Comment:
you need to check if the pointer is null to avoid a possible panic

Reply:
oof, nice catch.